src/config.js

```diff
@@ -321,6 +322,7 @@ export default class Config {
       this._cacheRootFolder = String(cacheRootFolder);
     }
     this.workspacesEnabled = this.getOption('workspaces-experimental') !== false;
+    this.workspacesNohoistEnabled = this.getOption('workspaces-nohoist-experimental') !== false;
```

Comment:
After thinking about this I think we don't really need to hide this feature behind a flag.
Workspaces is still behind "experimental" flag :)

Reply:
I am happy to remove it but it was requested specifically by @arcanis cited `workspaces-experimental` will be removed soon, just want to make sure we are all on the same page...

Reply:
ok, let's keep it